scripts/just.config.js

```diff
@@ -28,7 +28,7 @@ module.exports = function preset() {
   );
 
   task('depcheck', checkPublishing);
-  task('lint', eslintTask({ files: ['src/.'] }));
+  task('lint', eslintTask({ files: ['src/'] }));
```

Comment:
Is this the one line change that stops us from listing artifacts?

Reply:
Honestly, there were multiple issues and I ended up having to fix them all simultaneously. I think the `.` is what caused the issue, but I can't say 100%.